src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2015_Default.exe.sarif

```diff
@@ -76,7 +76,7 @@
             "id": "Warning_NativeWithInsecureStaticLibraryCompilands",
             "arguments": [
               "MixedMode_x64_VS2015_Default.exe",
-              "Microsoft (R) Optimizing Compiler : cxx : 19.0.24215.1 : [directly linked] [No hash value present] (.NETFramework,Version=v4.5.2.AssemblyAttributes.obj)\r\n"
+              "Microsoft (R) Optimizing Compiler : cxx : 19.0.24215.1 : [directly linked] [No hash value present] (.NETFramework,Version=v4.5.2.AssemblyAttributes.obj,stdafx.obj)\r\n"
```

Comment:
>stdafx.obj [](http://example.com/codeflow?start=167&length=10)

This is due to the 2nd in the PR description.
the object is `stdafx.obj`
it has PCH `stdafx.pch`
the source file without hash in it is `c:\program files (x86)\reference assemblies\microsoft\framework\.netframework\v4.5.2\system.dll`
not due to `stdafx.pch`.